Youtube Metadata(not the search) still uses old youtube API

The queryed metadata via http://gdata.youtube.com/feeds/api/videos/ is displaying the deprecation video, this does affects the wire outputs.

The new API seems to have a equivalent here: https://developers.google.com/youtube/v3/docs/videos/list

I tested, and worked fine but it gives the error:

[ERROR] addons/playx/lua/playx/providers/youtube.lua:128: attempt to index field
 'standard' (a nil value)
  1. onsuccess - addons/playx/lua/playx/providers/youtube.lua:128
   2. unknown - lua/includes/modules/http.lua:34

@SwadicalRag can you fix it?

--- a/lua/playx/providers/youtube.lua
+++ b/lua/playx/providers/youtube.lua
@@ -125,7 +125,7 @@ function YouTube.QueryMetadata(uri, callback, failCallback)
         --Thanks Wyozi/Wiox
         local length = tonumber(length_raw:match("(%d+)S") or 0) + tonumber(length_raw:match("(%d+)M") or 0)*60 + tonumber(length_raw:match("(%d+)H") or 0)*3600
         local tags = resultsTable.items[1].snippet.tags or {}
-        local thumbnail = resultsTable.items[1].snippet.thumbnails.standard.url
+        local thumbnail = resultsTable.items[1].snippet.thumbnails.default.url
         local comments = resultsTable.items[1].statistics.commentCount
 
         local faves = resultsTable.items[1].statistics.favoriteCount

